tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -525,13 +525,14 @@ def test_fsdp_mixed_with_sync(
 @pytest.mark.filterwarnings(r'ignore:.*The CUDA RNG state could not be loaded.*:UserWarning')
 @pytest.mark.filterwarnings(r'ignore:.*ShardedTensor.to only move tensor to its current device.*:UserWarning')
 @pytest.mark.filterwarnings(r'ignore:.*\(TP\) is experimental.*:FutureWarning')
+@pytest.mark.filterwarnings(r'ignore:.*The CUDA RNG state was saved with a different version of PyTorch.*')
```

Comment:
why was this change needed? we didn't change the checkpoints being used here right?